sampler.go

```diff
@@ -497,7 +513,7 @@ func (s *RemotelyControlledSampler) extractSampler(
 		if sampler, ok := s.sampler.(*adaptiveSampler); ok {
 			return sampler, strategies, nil
 		}
-		sampler, err := NewAdaptiveSampler(strategies)
+		sampler, err := NewAdaptiveSampler(strategies, defaultMaxOperations)
```

Comment:
it would be in good taste to have this optionally configurable. It might be time to change remote sampler constructor to use functional option pattern, as it has too many parameters already, not all of them required.